### PR TITLE
Bybit :: contractOrder :: add amount to precision

### PR DESCRIPTION
**Pull request by carlosmiei**
- related #13458

### CODE REVIEW

File: js/bybit.js [hunk 1]
```diff
@@ -2852,7 +2852,7 @@ module.exports = class bybit extends Exchange {
             'side': this.capitalize (side),
             'order_type': this.capitalize (type), // limit
             'time_in_force': 'GoodTillCancel', // ImmediateOrCancel, FillOrKill, PostOnly
-            'qty': amount,
+            'qty': this.amountToPrecision (symbol, amount),
```
Comment (kroitor) [inline js/bybit.js:2855]: Note that amountToPrecision returns a string, so we might need `parseInt` here.

Comment (carlosmiei) [reply js/bybit.js:2855]: @kroitor yeah but it is working with a string too 
image

Comment (carlosmiei) [reply js/bybit.js:2855]: @kroitor We can buy 2.5 of this contract, so I don't think we can use parseInt here, maybe leave it as is or parseFloat

Comment (kroitor) [reply js/bybit.js:2855]: @carlosmiei we usually do `parseFloat (amountToString (` in such cases, despite that it works with an undocumented type. On a sidenote, we can probably add a boolean flag whether toPrecision-methods should return a number or a string.

Comment (flieks) [reply js/bybit.js:2855]: but like BTCUSDT on usdt perpetual can have decimals like 0.001 BTC amount as qty. Then integer won't work right ?

Comment (carlosmiei) [reply js/bybit.js:2855]: @flieks BTC/USDT is a linear market so will be parsed as a float, check my last edit

**Pull request merged**